article/app/controllers/ArticleController.scala

```diff
@@ -161,7 +161,7 @@ object ArticleController extends Controller with RendersItemResponse with Loggin
 
   def renderArticle(path: String) = {
     LongCacheAction { implicit request =>
-      mapModel(path) {
+      mapModel(path, request.isEmail) {
```

Comment:
we only started using blocks for liveblogs, everything else uses the body.  it shouldn't be much work to move things over, but at the moment any article still in R2 will have no block.  Amp doesn't do anything special re blocks.